Jenkinsfile

```diff
@@ -166,6 +166,14 @@ pipeline {
         stage('Audit') {
           steps { sh 'ci/test rspec_audit'}
         }
+        stage('Policy Parser Gem') {
+          steps {
+            dir('gems/policy-parser') {
+              sh './test.sh'
+            }
+            junit 'gems/policy-parser/spec/reports/*.xml'
+          }
+        }
```

Comment:
why are we going to run this test as part of conjur repo ? is not it run in conjur-policy-parser repo either ? I think it should run only in one place